#Request
string status
int32 point_id

---
bool result
